### PR TITLE
set the same semver-range when installing peer deps.

### PR DESCRIPTION
**Pull request by andreineculau**
ref tobiipro/eslint-config-firecloud#19

### CODE REVIEW

File: npm-install-peer-dependencies [hunk 1]
```diff
@@ -55,6 +55,10 @@ node -e " \
               continue
         }
 
+        # cat <<< "$(jq ".devDependencies += {\"${NAME}\": \"${VSN}\"}" < package.json)" > package.json
+        cat <<< "$(npx json "this.devDependencies = this.devDependencies || {}" < package.json)" > package.json
```
Comment (IanSavchenko) [inline npm-install-peer-dependencies:59]: I can guess what it does, but you gonna need to explain to me how it actually works 😄

Comment (andreineculau) [reply npm-install-peer-dependencies:59]: here goes some "bashsplaining" :joy:

if you simplify the command : `cat <<< "$(foo < bar)" > bar`, you have

* `> bar` - write stdout to file
* `< bar` - read stdin from file
* `cat` - is cat
* `<<< ""` - is a oneline heredoc into stdin

in plain english:
 - output to stdout (via cat) the contents of the heredoc, and the heredoc is actually a command, and then into bar
- the key is the heredoc, because it doesn't stream, thus you buffer the command output, and only after you process the entire `package.json`, you start writing back to `package.json`

Comment (IanSavchenko) [reply npm-install-peer-dependencies:59]: Thank you a lot!

**Review by IanSavchenko — APPROVED** [review]

File: npm-install-peer-dependencies (hunk 1 above)
Comment (andreineculau) [reply npm-install-peer-dependencies:59]: actually `json` has an in-place flag. `jq` has a beard and knows better.
went for the less cryptic in-place version

https://github.com/tobiipro/eslint-config-firecloud/commit/12310e500a99bd88421fb9b30df4dc88f70c537c
